apps/src/code-studio/components/progress/StatusProgressDot.jsx

```diff
@@ -62,14 +66,31 @@ export const StatusProgressDot = React.createClass({
       }
     }
 
-    return (
-      <ProgressDot
-        level={level}
-        courseOverviewPage={courseOverviewPage}
-        stageId={stageId}
-        status={status}
-      />
-    );
+    if (experiments.isEnabled('progressBubbles')) {
+      const onCurrent = currentLevelId &&
+        ((level.ids && level.ids.map(id => id.toString()).indexOf(currentLevelId) !== -1) ||
+        level.uid === currentLevelId);
```

Comment:
What is `level.ids` vs `level.uid`?  Why does `currentLevelId` match both, and why do we need to check both (i.e. is `uid === currentLevelId` sufficient)?

Also, I thought we had [`Array.prototype.includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) polyfilled - it might be cleaner than `indexOf` here.